tests/browser/pages/relying-party.js

```diff
@@ -31,7 +31,7 @@ module.exports = class PlaywrightDevPage {
     const { searchParams } = new URL(this.page.url());
 
     return (
-      searchParams.get("client_id") && // FIXME: Restore checking of client_id
```

Comment:
Has this been fixed? The last time I tried restoring it, bugs were introduced in the testing.

Reply:
Adding this.TESTING_CLIENT_ID to the usages of the RelyingPartyPage constructor is what makes this work now.

Reply:
The browser tests pass with this change and the happy path test exercises this check - or is there something else I'm missing?